### ISSUE TITLE
GUI request

### ISSUE THREAD

**Issue by Systemhlp** [x64dbg/x64dbg#2735]
### Feature type

Debugging

### Describe the feature

Hello 
I think , it is better there is a window frame at disassembly portion.
Look below image.
![1](https://user-images.githubusercontent.com/87657997/133929556-46a659b5-5e12-4cab-9534-7d0259555273.png)
Thank you and good day !

**Comment by torusrxxx** [comment]
What do you actually want to do if the feature is implemented? I think if you want to inspect the code you can double click to go there.

**Comment by ZehMatt** [comment]
I think the author is asking to have a border/frame around the preview to better separate it.

**Comment by torusrxxx** [comment]
The default theme already has the border. Just dark theme has the issue.

**Comment by mrexodia** [comment]
Yeah, I ran into this issue while working on the dark theme with JustMagic.
Unfortunately I couldn’t find a workaround for it so I just left it like
this 🤷🏻‍♂️
